src/main/java/ulcrs/controllers/UlcController.java

```diff
@@ -0,0 +1,64 @@
+package ulcrs.controllers;
+
+import spark.Request;
+import spark.Response;
+import spark.RouteGroup;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+
+import static spark.Spark.before;
+import static spark.Spark.get;
+
+public class UlcController extends BaseController {
+
+    @Override
+    public RouteGroup routes() {
+        return () -> {
+            before("/*", (request, response) -> log.info("endpoint: " + request.pathInfo()));
+            get("/login", this::loginTest);
+        };
+    }
+
+    private String loginTest(Request request, Response response) {
```

Comment:
As a note, this function would be changing after merging, as we don't want the test function in our code. It can probably be left here for now though as reference, as it should be of little consequence.